plugins/chat/app/services/chat/publisher.rb

```diff
@@ -15,18 +15,22 @@ def self.thread_message_bus_channel(chat_channel_id, thread_id)
     end
 
     def self.calculate_publish_targets(channel, message)
-      targets =
-        if message.thread_om?
-          [
-            root_message_bus_channel(channel.id),
-            thread_message_bus_channel(channel.id, message.thread_id),
-          ]
-        elsif message.thread_reply?
-          [thread_message_bus_channel(channel.id, message.thread_id)]
-        else
-          [root_message_bus_channel(channel.id)]
-        end
-      targets
+      return [root_message_bus_channel(channel.id)] if !allow_publish_to_thread?(channel)
+
+      if message.thread_om?
```

Comment:
not a great name perhaps a future rename is in order

Reply:
OM stands for "original message", maybe I could just expand it to `thread_original_message?`